general-concepts/dependencies/text.xml

```diff
@@ -461,6 +461,12 @@ DEPEND="!build? (
 <title>Any of Many Dependencies</title>
 <body>
 
+<note>
+To ease dependency resolution for Portage, it is recommended that you sort
+the atoms in <e>preferred</e> order first. See
```

Comment:
No "atoms" please. We're trying to abandon this term ever since [bug 174322](https://bugs.gentoo.org/174322) (with mixed success, I must admit).

Reply:
Also, more specifically, [bug 603970](https://bugs.gentoo.org/603970) for the devmanual.

Reply:
Ah, I should've remembered that. I'm not sure this is realistic until Portage itself doesn't use 'atom' in its output though. That's exactly where I picked it up.